tests/integration/utils/flow_tools.py

```diff
@@ -39,15 +39,27 @@
 from .flow_writer import flow_config_str
 
 
-def _make_src_flow(tmp_path, conf):
+def _make_src_flow(src_path, conf=None):
     """Construct a workflow on the filesystem"""
-    flow_run_dir = (tmp_path / str(uuid1()))
-    flow_run_dir.mkdir(parents=True, exist_ok=True)
+    if conf is None:
+        conf = {
+            'scheduler': {
+                'allow implicit tasks': True
+            },
+            'scheduling': {
+                'graph': {
+                    'R1': 'one'
+                }
+            }
+        }
```

Comment:
This is a copy of `one_conf`?

I don't think we need an implicit configuration?